components/Carousel/Carousel.stories.tsx

```diff
@@ -98,3 +114,26 @@ export const MultipleCardsCarousel: StoryObj<MultipleCardsProps> = {
     cardCount: 3,
   },
 };
+
+export const FullScreenCarousel = () => {
+  return (
+    <Carousel isFullWidth>
+      {mockInnerImages.map((image, index) => {
+        return (
+          <div className="aspect-video rounded lg:max-w-4xl" key={index}>
```

Comment:
Is it possible to move these style to the image itself?, that way to get rid of the `div` element.
Also, a `mx-auto` would be good to center the image!

Reply:
@TonyMckes actually it works! Thank you 😊